code/modules/events/tear.dm

```diff
@@ -1,47 +1,101 @@
+			C.flicker()
+
+/datum/event/tear/proc/spawn_tear(location)
+	TE = new /obj/effect/tear(location)
+
+/datum/event/tear/announce()
+	GLOB.event_announcement.Announce("A tear in the fabric of space and time has opened. Expected location: [impact_area.name].", "Anomaly Alert", 'sound/AI/anomaly.ogg')
 
 /datum/event/tear/end()
 	if(TE)
 		qdel(TE)
 
+// The portal
 /obj/effect/tear
-	name="Dimensional Tear"
-	desc="A tear in the dimensional fabric of space and time."
-	icon='icons/effects/tear.dmi'
-	icon_state="tear"
+	name = "dimensional tear"
+	desc = "A tear in the dimensional fabric of space and time."
+	icon = 'icons/effects/tear.dmi'
+	icon_state = "tear"
 	density = 0
 	anchored = 1
 	light_range = 3
 
+	// Huge sprite, we shift it to make it look more natural.
+	pixel_x = -106
+	pixel_y = -96
+
 /obj/effect/tear/Initialize(mapload)
 	. = ..()
+	// Sound cue to warn people nearby.
+	playsound(get_turf(src), 'sound/magic/drum_heartbeat.ogg', 100)
+
+	// Portal opening animation.
 	var/atom/movable/overlay/animation = new(loc)
+	animation.pixel_x = pixel_x
+	animation.pixel_y = pixel_y
 	animation.icon_state = "newtear"
 	animation.icon = 'icons/effects/tear.dmi'
 	animation.master = src
 	QDEL_IN(animation, 1.5 SECONDS)
-	addtimer(CALLBACK(src, .proc/spew_critters), rand(30, 120))
 
-/obj/effect/tear/proc/spew_critters()
+	// We spawn the minions first, then the boss.
+	addtimer(CALLBACK(src, .proc/spawn_mobs), 2 SECONDS)
+	addtimer(CALLBACK(src, .proc/spawn_leader), 5 SECONDS)
+
+/obj/effect/tear/proc/spawn_mobs()
+	var/list/possible_mobs = list(
+		/mob/living/simple_animal/hostile/netherworld,
+		/mob/living/simple_animal/hostile/netherworld/migo,
+		/mob/living/simple_animal/hostile/faithless)
+
+	// We break some of those flickering consoles from earlier.
+	// Mirrors as well, for the extra bad luck.
+	for(var/obj/machinery/computer/C in range(6, src))
+		C.obj_break()
+	for(var/obj/structure/mirror/M in range(6, src))
+		M.obj_break()
+
+	// Spawning mobs.
 	for(var/i in 1 to 5)
-		var/mob/living/simple_animal/S
-		S = create_random_mob(get_turf(src), HOSTILE_SPAWN)
-		S.faction |= "chemicalsummon"
-		if(prob(50))
-			for(var/j = 1, j <= rand(1, 3), j++)
-				step(S, pick(NORTH, SOUTH, EAST, WEST))
+		var/chosen_mob = pick(possible_mobs)
+		var/mob/M = new chosen_mob(loc)
+		M.faction = list("rift")
+		step(M, pick(GLOB.cardinal))
+
+// We spawn a leader mob to make the portal actually dangerous.
+/obj/effect/tear/proc/spawn_leader()
+	var/mob/living/simple_animal/M = new /mob/living/simple_animal/hostile/hellhound(get_turf(src))
+	M.name = "frenzied hellhound" 	// Rename it to indicate that their stats are different from the regular one.
+	M.melee_damage_lower = 30		// Heavy melee damage to encourage the usage of guns.
+	M.melee_damage_upper = 50
+	M.maxHealth = 300				// The minions are low health, hard hitting creatures; the leader should stand till the end.
+	M.health = 300
+	M.faction = list("rift")
```

Comment:
please make a subtype of the mob and new that instead.

Reply:
done in 43f256270113cd432155f337a7faeaa855fdcbea